app/controllers/subscriptions_controller.rb

```diff
@@ -12,7 +12,9 @@ def new
   def create
     subscription = Subscription.new(daily_subscription_params)
     @subscription = SubscriptionPresenter.new(subscription)
+    recaptcha_valid = verify_recaptcha(model: subscription, action: 'subscription')
```

Comment:
If this is Boolean, can we denote this with a `?`?

Reply:
Good catch.  We can and will do.